src/content/docs/logs/ui-data/lookup-tables-ui.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: "Upload CSV-format lookup tables"
+metaDescription: 'In the New Relic logs UI, upload CSV-format lookup tables and use that data in combination with other New Relic data.'
+---
+
+import logsLookupTableUi from 'images/logs_screenshot-crop_lookup-table-ui.webp'
+
+Our **lookups** feature lets you enrich your log data, and other New Relic-stored data, with data about your business that you define in a CSV file. 
+
+## Why use lookups? [#overview]
+
+You can use lookups to make your telemetry data easier to understand, group data in custom ways, and include more information that wasn't available at the time for previously reported data. From the logs UI, you can upload a CSV file and use the `lookup()` function to include the lookup data in a NRQL query. 
```

Comment:
"that wasn't available at the time" seems unclear; at what time? How about, "...and include additional information that wasn't previously reported"? [Same as below]